.github/workflows/manual-deploy-testnet-l1.yml

```diff
@@ -96,3 +98,8 @@ jobs:
           ports: '8025 8026 9000 9001'
           cpu: 2
           memory: 8
+
+      - name: 'Dispatch trigger'
+        if: ${{ github.event_name == 'schedule' }}
```

Comment:
Note by default this if condition will only trigger on a successful deployment (i.e you dont need to add an extra condition on if success())